### DISCUSSION TITLE
Mapsforge on Ubuntu22

### DISCUSSION THREAD

**Discussion by ozan123456789** [mapsforge/mapsforge#1351] [Q&A] [answered]
I am trying to open Mapsforge on Ubuntu with android studio but I am getting errors like below:
![Screenshot from 2022-07-30 00-35-23](https://user-images.githubusercontent.com/29802498/181846298-605dba27-4a87-4061-8799-72739cb72295.png)

My SDK and JDK settings:
![Screenshot from 2022-07-30 00-40-37](https://user-images.githubusercontent.com/29802498/181846955-c694c91d-497b-4ec3-97fc-81c2f50a16bc.png)

Android Studio version:
![Screenshot from 2022-07-30 00-42-45](https://user-images.githubusercontent.com/29802498/181846956-0d3d3b6b-b1fa-4890-b4d7-7ce90447b7fd.png)
What cause of this and how I can fix it.

**Comment by devemux86** [comment]
The [main page](https://github.com/mapsforge/mapsforge) mentions:

_"Building Mapsforge: set `ANDROID_HOME` environment variable with the Android SDK installation directory"_

https://developer.android.com/studio/command-line/variables

See also: https://github.com/mapsforge/mapsforge/discussions/1291

**Reply by ozan123456789** [reply]
I set ANDROID_HOME and JAVA_HOME but result is same.

![Screenshot from 2022-07-30 11-58-00](https://user-images.githubusercontent.com/29802498/181903243-ccc6568b-a09f-4486-b0b1-19fc5487cf9e.png)

**Comment by devemux86** [comment, marked as answer]
https://github.com/mapsforge/mapsforge/discussions/1291#discussioncomment-2159103

`ANDROID_HOME` may be set in `.profile` (and log out).

If you use the IDE via snap,
reinstall the IDE from tar package downloaded from IDE website, works better.
